node/src/main/java/eu/cloudnetservice/node/service/defaults/AbstractService.java

```diff
@@ -766,6 +771,11 @@ protected void prepareService() {
       configuration.privateKeyPath() == null ? null : wrapperDir.resolve("privateKey"));
   }
 
+  protected void downloadInclusionFiles(@NonNull GetRequest request, @NonNull Path destination) {
```

Comment:
Is it indeted that the method name is plural but really only downloads one file?